src/v3/src/components/Widget/style.module.css

```diff
@@ -10,7 +10,7 @@
   visibility: hidden;
 }
 :global(.hide) {
-  display: none;
+  display: none !important;
```

Comment:
instead of adding `!important` here, could you define a module level `.hide` css class in `src/v3/src/components/AuthContainer/style.module.css`?

Unless there is some sort of benefit to doing it this way vs the module level way? I only suggested this because I am not sure of the ramifications of changing this css since the classes here are mostly used by Okta hosted JSP.